docs/atomic-install.1.md

```diff
@@ -77,6 +77,10 @@ Installing a system container consists of checking it the image by
 default under /var/lib/containers/atomic/ and generating the
 configuration files for runc and systemd.
 
+**--user**
+If running as non-root, specify to install the image from the current
+OSTree repository and manage it through systemd and bubblewrap.
```

Comment:
I think it is better that for the `install`command the user is explicit on what kind of container is going to be installed.  We will be able in future to support other kind of containers as well.